backend/typescript/middlewares/validators/schedulingValidators.ts

```diff
@@ -34,6 +34,9 @@ export const createSchedulingDtoValidator = async (
       .status(400)
       .send(getApiValidationError("pickupLocation", "string"));
   }
+  if (!validatePrimitive(req.body.dayPart, "string")) {
```

Comment:
is `dayPart` always going to be either "morning", "afternoon", or "evening"? In that case, should we constrain the type to be an enum?

Reply:
I asked @Shehryar21 if I should change it to an enum and he said it'll make it easier to accommodate changes as a string

Reply:
Updated both `dayPart` and `frequency` to be enums